frontend/src/components/ScheduleCreation.vue

```diff
@@ -137,7 +144,9 @@ onMounted(() => {
   generateZoomLink.value = scheduleInput.value.meeting_link_provider === MeetingLinkProviderType.Zoom;
 
   // Set a new reference
-  referenceSchedule.value = { ...scheduleInput.value };
+  // Sidenote: I'm too dumb to properly clone this Object without referencing the same availability array. So for now
+  // this is the good ol' JSON workaround (structuredClone, Object assign or spread won't work here)
+  referenceSchedule.value = JSON.parse(JSON.stringify(scheduleInput.value));
```

Comment:
I had a lot of issues due to the availabilities array being referenced from all over the components. The only thing that helped was the JSON workaround. Let me know, if you have a more proper way to deep clone that object 😇

Reply:
Naw that's fine. Feel free to remove the comments, and just mentioned deep clone.